test/Microsoft.ML.Functional.Tests/ModelFiles.cs

```diff
@@ -213,6 +213,38 @@ void AssertIsGam(ITransformer trans)
             Done();
         }
 
+        public class ModelInput
+        {
+#pragma warning disable SA1401
+            public string[] CategoricalFeatures;
+            public float[] NumericalFeatures;
+#pragma warning restore SA1401
+        }
+
+        public class ModelOutput
+        {
+#pragma warning disable SA1401
+            public float[] Score;
+#pragma warning restore SA1401
+        }
+
+
+        [Fact]
+        public void LoadModelWithOptionalColumnTransform()
+        {
+            SchemaDefinition inputSchemaDefinition = SchemaDefinition.Create(typeof(ModelInput));
+            inputSchemaDefinition[nameof(ModelInput.CategoricalFeatures)].ColumnType = new VectorDataViewType(TextDataViewType.Instance, 5);
+            inputSchemaDefinition[nameof(ModelInput.NumericalFeatures)].ColumnType = new VectorDataViewType(NumberDataViewType.Single, 3);
+            var mlContext = new MLContext();
+            ITransformer trainedModel;
+            DataViewSchema dataViewSchema;
+            trainedModel = mlContext.Model.Load(Path.Combine(Directory.GetCurrentDirectory(), @"../../../../test/data/backcompat/modelwithoptionalcolumntransform.zip"), out dataViewSchema);
```

Comment:
>Combine [](start = 53, length = 7)

There is a `GetDataPath` method in the base class:
`GetDataPath("backcompat", "modelwithoptionalcolumntransform.zip")`